software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessWinRmDriver.java

```diff
@@ -161,12 +161,16 @@ public WinRmMachineLocation getMachine() {
     }
 
     protected int executeCommandInTask(String command, String psCommand, String phase) {
+        return executeCommandInTask(command, psCommand, phase, false);
+    }
+
+    protected int executeCommandInTask(String command, String psCommand, String phase, boolean suppressFailures) {
         return newScript(phase)
                 .setCommand(command)
                 .setPsCommand(psCommand)
                 .failOnNonZeroResultCode()
                 .gatherOutput()
-                .execute();
+                .execute(suppressFailures);
```

Comment:
Why is `suppressFailures` special (i.e. passing it as a parameter to `execute()`, rather than calling `.suppressFailures(suppressFailures)` to configure the script before calling a no-arg `execute()`?